src/bigint.rs

```diff
@@ -1578,6 +1649,49 @@ impl BigInt {
 
     /// Creates and initializes a `BigInt`.
     ///
+    /// The bytes are in big-endian byte order
+    ///
+    /// # Examples
+    ///
+    /// ```
+    /// use num::bigint::{BigInt, Sign};
+    ///
+    /// assert_eq!(BigInt::from_bytes_be(Sign::Plus, "A".as_bytes()),
+    ///            BigInt::parse_bytes("65".as_bytes(), 10).unwrap());
+    /// assert_eq!(BigInt::from_bytes_be(Sign::Plus, "AA".as_bytes()),
+    ///            BigInt::parse_bytes("16705".as_bytes(), 10).unwrap());
+    /// assert_eq!(BigInt::from_bytes_be(Sign::Plus, "AB".as_bytes()),
+    ///            BigInt::parse_bytes("16706".as_bytes(), 10).unwrap());
+    /// assert_eq!(BigInt::from_bytes_be(Sign::Plus, "Hello world!".as_bytes()),
+    ///            BigInt::parse_bytes("22405534230753963835153736737".as_bytes(), 10).unwrap());
+    /// ```
+    #[inline]
+    pub fn from_bytes_be(sign: Sign, bytes: &[u8]) -> BigInt {
+        BigInt::from_biguint(sign, BigUint::from_bytes_be(bytes))
+    }
+
+    /// Creates and initializes a `BigInt`.
+    ///
+    /// The bytes are in little-endian byte order
+    #[inline]
+    pub fn from_bytes_le(sign: Sign, bytes: &[u8]) -> BigInt {
+        BigInt::from_biguint(sign, BigUint::from_bytes_le(bytes))
+    }
+
+    /// Returns the byte representation of the `BigInt` in little-endian byte order
```

Comment:
Actually, I'd prefer this to simply return the byte representation regardless of the sign. If the user wants the underlying bytes representing the integer, they should be allowed to get them. If they need to handle negative numbers a certain way, they can still access that information from the original `BigInt` and handle that case accordingly.